src/renderers/webgl/WebGLTextures.js

```diff
@@ -1594,7 +1594,12 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 				}
 
 				_gl.blitFramebuffer( 0, 0, width, height, 0, 0, width, height, mask, _gl.NEAREST );
-				_gl.invalidateFramebuffer( _gl.READ_FRAMEBUFFER, invalidationArray );
+
+				if ( ! renderTarget.skipInvalidateFramebuffer ) {
+
+					_gl.invalidateFramebuffer( _gl.READ_FRAMEBUFFER, invalidationArray );
+
+				}
```

Comment:
Can we check if the UA is Oculus Browser at init time and do this instead?

```js
if ( isOculusBrowser ) {

	_gl.invalidateFramebuffer( _gl.READ_FRAMEBUFFER, invalidationArray );

}
```

Reply:
That will just make it faster for Oculus browser and not other mobile GPUs like Hololens.

Reply:
Ah, right...

Well, what I'm proposing is to make this automatic without the user having to know about it.

You said:

> Some WebGL implementations (Edge and Safari on M1) don't handle invalidateFramebuffer correctly.

But https://github.com/mrdoob/three.js/issues/22967#issuecomment-1018315477 says Chrome on Android is affected too.

Is there any way of making a UA check and have this code for 3-4 months?